CONTRIBUTING.md

```diff
@@ -52,6 +54,8 @@ The Minimal Supported Rust Version is 1.22.0 (enforced by our Travis).
 Commits should cover both issues fixed and solutions' rationale.
 These [guidelines](https://chris.beams.io/posts/git-commit/) should be kept in mind.
 
+To facilitate communication with other contributors, if you are working on a GitHub Issue please assign it to yourself using GitHub's "assignee" field.
```

Comment:
I don't actually know if non-"triage"-permissions folks can do this. Maybe just note that you should first check that no one is assigned and then comment suggesting that you're working on it, but don't hesitate to ask if the assigned party or previous commenters are still working on it if its been a while.

Reply:
Fixed